### PR TITLE
adds start command, updates readme and improves /tldr

### PR DESCRIPTION
**Pull request by Hoi15A**
/tldr now no longer requires regex to understand /tldr@botname <args>
/start now replies with a greeting and some examples
the readme now includes the new BOT_USERNAME env
variable

### CODE REVIEW

**Comment by fuerbringer** [comment]
Cloning your branch showed that the `/start` command works. However the command `/tldr@<botusername> <platform>/<page>` seems to be broken again (while plain `/tldr <platform>/<page>` works):

scrot

My `.env` had `BOT_USERNAME=tldr_pages_bot` defined.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This bot can be run with Docker or simply `npm start`. In both cases `.env` need
 ```
 NODE_ENV=production
 BOT_TOKEN=<TELEGRAMBOTFATHERTOKEN>
+BOT_USERNAME=<TELEGRAMBOTUSERNAME>
```
Comment (fuerbringer) [inline README.md:15]: So why is this needed now? It could have previously been read from `bot.options.username = botInfo.username`.

**Review by fuerbringer — CHANGES_REQUESTED** [review]
See feedback above.

File: README.md (hunk 1 above)
Comment (Hoi15A) [reply README.md:15]: It is required because the information is required for creating the bot instance before you can even access `botInfo.username`

Comment (fuerbringer) [reply README.md:15]: Alright, I guess that's what's meant here in the documentation: http://telegraf.js.org/#/?id=command-handling-in-group

Comment (Hoi15A) [reply README.md:15]: correct

**Comment by Hoi15A** [comment]
As I already mentioned in telegram, `/command@bot` is really designed for group chats to be able to differentiate commands (especially if the same command exists 2x). This format is not really expected in a private chat though as you have found out due to it no longer working.

We could jump through some hoops to get it to work, but in my opinion its not worth the time fiddling around with getting the group format working in private chat as it will likely never be used anyways.

**Comment by fuerbringer** [comment]
All good, thanks for the clarifications.

**Review by fuerbringer — APPROVED** [review]
